be/src/common/daemon.cpp

```diff
@@ -75,22 +79,22 @@ class ReleaseColumnPool {
     size_t _freed_bytes = 0;
 };
 
-void gc_tcmalloc_memory(void* arg_this) {
+void gc_memory(void* arg_this) {
     using namespace starrocks::vectorized;
     const static float kFreeRatio = 0.5;
     GCHelper gch(config::tc_gc_period, config::memory_maintenance_sleep_time_s, MonoTime::Now());
 
     Daemon* daemon = static_cast<Daemon*>(arg_this);
     while (!daemon->stopped()) {
         sleep(config::memory_maintenance_sleep_time_s);
-#if !defined(ADDRESS_SANITIZER) && !defined(LEAK_SANITIZER) && !defined(THREAD_SANITIZER)
+#if !defined(ADDRESS_SANITIZER) && !defined(LEAK_SANITIZER) && !defined(THREAD_SANITIZER) && !defined(USE_JEMALLOC)
         MallocExtension::instance()->MarkThreadBusy();
 #endif
         ReleaseColumnPool releaser(kFreeRatio);
         ForEach<ColumnPoolList>(releaser);
         LOG_IF(INFO, releaser.freed_bytes() > 0) << "Released " << releaser.freed_bytes() << " bytes from column pool";
 
-#if !defined(ADDRESS_SANITIZER) && !defined(LEAK_SANITIZER) && !defined(THREAD_SANITIZER)
+#if !defined(ADDRESS_SANITIZER) && !defined(LEAK_SANITIZER) && !defined(THREAD_SANITIZER) && !defined(USE_JEMALLOC)
```

Comment:
Does jemalloc need a external gc thread?

Reply:
> Does jemalloc need a external gc thread?

It has dedicated threads internally responsible for purge, we do not need to create additional